tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -71,8 +66,8 @@ struct testing_callbacks {
     void _On_dynamic_precision(_Auto_id_tag) {
         assert(expected_auto_dynamic_precision);
     }
-    void _On_conversion_spec(CharT mod, CharT type) {
-        assert(static_cast<char>(mod) == expected_chrono_specs[curr_index]._Modifier);
+    void _On_conversion_spec(char mod, CharT type) {

```

Comment:
Similar to the other comment, I believe this will be called with `CharT`, so it should take that and then internally perform narrowing, so I'll leave this as-is.